liquidity-module/index.md

```diff
@@ -303,73 +411,80 @@ Add `username`s mnemonic passphrase and hit enter. You will see something simila
 
 ## Create a Liquidity Pool
 
-To create a liquidity pool, enter the following command:
-Make sure to replace `longibchash` with the hash denom you received on the previous step. Replace `username` with your account username.
+Run the `liquidity create-pool` command to create a liquidity pool with two tokens: 1100000uphoton and 1500000ibc/denomhash. 
+
+Make sure to replace `ibc/denomhash` with the hash denom you received on the previous step and that you can verify in the [testnet supply](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/supply). 
+
+Replace `username` with your account username.
 
 ```bash
-gaiad tx liquidity create-pool 1 1100000uphoton,1500000ibc/longibchash --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 --gas 2000000 -y
+gaiad tx liquidity create-pool 1 1100000uphoton,1500000ibc/denomhash --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 --gas 2000000 -y
 ```
-**Optional:** Add -y flag to bypass confirmation prompt
 
-Confirm the pool has been created. See:
+The optional `-y` flag specifies to bypass the confirmation prompt. 
+
+Confirm the pool has been created. See the pools endpoint:
 
 [https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools](https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools)
 
 
-## Swap Token
+## Swap Tokens
 
 You are ready to swap tokens! You now have uphoton token in your account and want to swap for the new IBC coin:
 
 ```bash
-gaiad tx liquidity swap 1 1 100000uphoton ibc/longibchash 0.1 0.003 --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443 -y
```

Comment:
It will be helpful to add this line to provide which arguments are needed. Also, write a line that explains which `pool-id` they should use.
> gaiad tx liquidity swap [pool-id] [swap-type] [offer-coin] [demand-coin-denom] [order-price] [swap-fee-rate]`